runtime/gc_realtime/RealtimeGC.cpp

```diff
@@ -1336,111 +730,72 @@ MM_RealtimeGC::condYield(MM_EnvironmentBase *env, U_64 timeSlackNanoSec)
 bool
 MM_RealtimeGC::isMarked(void *objectPtr)
 {
-	return _markingScheme->isMarked(static_cast<J9Object*>(objectPtr));
+	return _markingScheme->isMarked((omrobjectptr_t)(objectPtr));
```

Comment:
Even in this form cast would not be necessary if input parameter type is changed from void* to omrobjectptr_t*
It can be done later in cleanup